tests/tensors/backend_test.py

```diff
@@ -0,0 +1,22 @@
+# pylint: disable=no-self-use,invalid-name
+import numpy
+from numpy.testing import assert_allclose
+from keras import backend as K
+
+from deep_qa.tensors.backend import cumulative_sum
+from ..common.test_case import DeepQaTestCase
+
+
+class TestBackendTensorFunctions(DeepQaTestCase):
+    def test_cumulative_sum(self):
```

Comment:
I'm not so sure it's necessary there, as the mask is not used in the computation, and we just return one of the input masks.